### PR TITLE
Add workspace notebook command

### PR DESCRIPTION
**Pull request by Ladicle**

### CODE REVIEW

File: internal/workspaces/notebooks/notebooks.go [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+		imgType string
+	)
+	cmd := &cobra.Command{
+		Use:  "create",
+		Args: cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return create(cmd.Context(), name, imgType)
+		},
+	}
+	cmd.Flags().StringVar(&name, "name", "", "Name of the API key")
```
Comment (kkaneda) [inline internal/workspaces/notebooks/notebooks.go:49]: Name of the notebook?

File: internal/workspaces/notebooks/notebooks.go [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+		id string
+	)
+	cmd := &cobra.Command{
+		Use:  "get",
+		Args: cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return get(cmd.Context(), id)
+		},
+	}
+	cmd.Flags().StringVar(&id, "id", "", "ID of the notebook")
```
Comment (kkaneda) [inline internal/workspaces/notebooks/notebooks.go:76]: No need to change now, but we probably should support lookup with `name`. The backend also probably have `name` as DB column and have a unique constraint on `(name, projectID)`?

**Review by kkaneda — APPROVED** [review]
LG!

File: internal/workspaces/notebooks/notebooks.go [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+	var req jv1.GetNotebookRequest
+	var resp jv1.Notebook
+	if err := ihttp.NewClient(env).Send(http.MethodGet, fmt.Sprintf("%s/%s", path, id), &req, &resp); err != nil {
+		return err
+	}
+	return printNotebook(&resp)
+}
+
+func printNotebook(nb *jv1.Notebook) error {
+	b, err := json.MarshalIndent(&nb, "", "    ")
```
Comment (kkaneda) [inline internal/workspaces/notebooks/notebooks.go:155]: I did this for job, but it was just a quick hack. Any alternative idea appreciated!

File: internal/workspaces/notebooks/notebooks.go (hunk 2 above)
Comment (kenji-cloudnatix) [reply internal/workspaces/notebooks/notebooks.go:76]: Btw, fine-tuning jobs don't have names as it is not in the OpenAI API spec, but I'm wondering if we should have that somehow.

Comment (Ladicle) [reply internal/workspaces/notebooks/notebooks.go:76]: I feel thinking about the name every time is a bit annoying as a job tends to have a short lifetime. That said an optional name or description would be useful.

Comment (kenji-cloudnatix) [reply internal/workspaces/notebooks/notebooks.go:76]: Agreed!

**Pull request merged**